### PR TITLE
[CA-982] chore(packages): update latest and renamed packages

### PR DESCRIPTION
**Pull request by stkaratz**
Updates after renaming:
- Configit.Ace.Xml.Import.Client to Configit.Ace.Model.Import.Client
- Configit.Ace.Model.Api.Client to Configit.Ace.Model.Client

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -8,18 +7,18 @@ Welcome to the Configit SDK, home of libraries and sample code for Configit part
 
 The following libraries are available for Configit Ace.
 
-### Ace Model Client for .NET       
+### Ace Model Client for .NET
 
-A helper library for the Model API, letting you read or write data Ace Model data.
+A helper library for the Ace Model API, letting you read or write Ace Model data.
```
Comment (moving-northwards) [inline profile/README.md:12]:
```suggestion
A helper library for the Ace Model API, letting you read and write Ace Model data.
```

File: profile/README.md [hunk 2]
```diff
@@ -8,18 +7,18 @@ Welcome to the Configit SDK, home of libraries and sample code for Configit part
```
Comment (moving-northwards) [inline profile/README.md:7]: I don't know why I can't make a suggestion comment here but I would move the whole second sentence onto the next line if possible. 

> **Getting access**: The Configit SDK repositories and NuGet/npm packages are private. 
> To request access, mail configit-sdk-req@configit.com.

**Review by moving-northwards — APPROVED** [review]
I left a couple of suggestions.

**Comment by stkaratz** [comment]
> I left a couple of suggestions.

Cool thanks @moving-northwards. I also wanted a second pair of eyes to take a look.

**Pull request merged**